tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/mailet/OpenPaasAmqpForwardAttribute.adoc

```diff
@@ -0,0 +1,28 @@
+=== OpenPaasAmqpForwardAttribute
```

Comment:
Sounds like a aweful location for doc. Why not /docs folder?

Reply:
I was following a similar approach to the AIBotMailet.

Reply:
Please move the `/docs` folder.

In this please nobody will ever find it.

Reply:
Okay, but it is not clear in which subfolder it should be placed.

Reply:
Move it onto https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/pages/tmail-backend/configure/index.adoc

Let's move on. 

We will split this file better into https://github.com/linagora/tmail-backend/issues/1295

Reply:
Ok looks like it is already done

Just delete this duplicated file please